### PR TITLE
35: add jolokia support for monitoring

### PR DESCRIPTION
**Pull request by ghost**
Jolokia was added for monitoring purposes. #35

### CODE REVIEW

**Comment by ehartung** [comment]
:+1:

**Comment by lmontrieux** [comment]
Thanks for the PR! We'll be looking into it as soon as possible.

**Comment by adyach** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -41,14 +41,17 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && apt-get purge -y --auto-remove $BUILD_DEPS \
     && apt-get autoremove -y \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /root/.m2
+    && rm -rf /var/lib/apt/lists/* /root/.m2 \
+
+    # jolokia
+    && wget -q -O /opt/jolokia-jvm-1.3.7-agent.jar "http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.3.7/jolokia-jvm-1.3.7-agent.jar"
```
Comment (lmontrieux) [inline Dockerfile]: Can we get this over https instead?

Comment (lmontrieux) [reply Dockerfile]: Even better, would it be possible to use an ENV for the version number?

Comment (ghost) [reply Dockerfile]: @lmontrieux there's no way to get .jar over https. I updated it to use ENV.

**Comment by lmontrieux** [comment]
👍

**Comment by rcillo** [comment]
👍

**Pull request merged**